packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0]
+
+### Changed
+
+- fix: notification services controller mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Comment:
```suggestion
### Fixed

- `NotificationServicesController` mobile fixes ([#4441](https://github.com/MetaMask/core/pull/4441))
```

Reply:
TY, new to this repo!

Reply:
Yep good callout, I've included the Controller name, to make it a bit more explicit on the changes.